### PR TITLE
fix broken links in yum install

### PR DESCRIPTION
**Pull request by Loquacity**
# Description

Two xrefs in the yum install page were broken. This PR updates them to the correct pages.

# Version

Which documentation version does this PR apply to?

- [x] Latest (Default)
- [ ] Version 1.7
- [ ] Older [specify]

# Links

Fixes https://github.com/timescale/docs/issues/76

### CODE REVIEW

**Review by ryanbooz — APPROVED** [review]

File: timescaledb/how-to-guides/install-timescaledb/installation-yum.md [hunk 1]
```diff
@@ -82,4 +82,7 @@ sudo -u postgres service postgres-13 start
 [pgdg]: https://yum.postgresql.org/repopackages.php
 [config]: /how-to-guides/configuration/
 [createuser]: https://www.postgresql.org/docs/current/sql-createrole.html
+[contact]: https://www.timescale.com/contact
+[slack]: https://slack.timescale.com/
+[multi-node-basic]: /how-to-guides/install-timescaledb/post-install-setup/-multi-node-basic
```
Comment (ryanbooz) [inline timescaledb/how-to-guides/install-timescaledb/installation-yum.md:87]: I'm confused... why are these here?

**Comment by Loquacity** [comment]
It would appear as though the ground as shifted dramatically under this PR, and obliterated the actual changes. I'm going to kill this PR, but mark the issue as resolved.
